1.generate-profiles/profiling_config.yaml

```diff
@@ -64,3 +65,31 @@ aggregate:
     guide:
       - Metadata_Foci_Barcode_MatchedTo_GeneCode
       - Metadata_Foci_Barcode_MatchedTo_Barcode
+---
+normalize:
+  perform: true
+  output_basedir: data/profiles
+  method: standardize
+  levels:
+    - gene
+    - guide
+    - single_cell
+  by_samples: all
+  features: infer
+---
+feature_select:
+  perform: true
+  output_basedir: data/profiles
```

Comment:
Pointing out that here as in the [normalize][output_basedir] (mentioned in my review @16) the output_basedir is a nested folder and we have up until now used only single folders, I believe. 
Again, not necessarily a problem, just pointing it out.

Reply:
Noting also here the discussion we had in https://github.com/broadinstitute/pooled-cell-painting-profiling-recipe/pull/16#discussion_r432608560